### DISCUSSION TITLE
Unable to import the coordinates file in Neper

### DISCUSSION THREAD

**Discussion by kumaranvenk** [neperfepx/neper#784] [Q&A]
I used the following code: neper -T -n 164 -dim 2 -domain "square(200,200) 
-morphooptiini "coo:file(coo1),weight:file(wts)", but the code is getting aborted

some data from coo1 file:
1369.45	424.162
477.995	562.569
822.255	468.65

some data from wts file:
6.7703
29.6186
14.7555
27.4547
8.8129

please suggest to overcome the problem

**Comment by rquey** [comment]
Some response:
- There was indeed a bug, fixed in 4.7.1-6.
- All the points provided in coo1 are outside the domain, which is probably not what you want.

**Reply by kumaranvenk** [reply]
how can I restrict it within the domain, kindly suggest

**Reply by kumaranvenk** [reply]
I have updated the latest available version 4.7.0. I plotted for 12 seed points, can you suggest how to restrict the domain in above case, because, when I am trying to plot, blank images are coming.

The code I used: neper -T -n 164 -dim 2 -domain "square(800,800) -morphooptiin"coo:file(coo),weight:file(wts)" -statcell "area"


some data from coo1 file:
1369.45 424.162
477.995 562.569
822.255 468.65

some data from wts file:
6.7703
29.6186
14.7555
27.4547
8.8129
